components/lang/pom.xml

```diff
@@ -201,7 +218,7 @@
                         </goals>
                         <configuration>
                             <createDependencyReducedPom>true</createDependencyReducedPom>
-                            <minimizeJar>true</minimizeJar>
+                            <!--<minimizeJar>true</minimizeJar>-->
```

Comment:
Do we need to remove this ? If this temporary please add a TODO and a small explanation